datasetinsights/io/tracker/mlflow.py

```diff
@@ -45,31 +45,40 @@ class MLFlowTracker:
     EXP_NAME = "experiment"
     RUN_NAME = "run"
     DEFAULT_RUN_NAME = "run-" + TIMESTAMP_SUFFIX
+    TRACKER = "tracker"
+    MLFLOW_TRACKER = "mlflow"
+    DEFAULT_EXP_NAME = "datasetinsights"
 
-    def __init__(self, mlflow_config):
+    def __init__(
+        self,
+        *,
+        client_id=None,
+        host=None,
+        run=DEFAULT_RUN_NAME,
```

Comment:
Why not call this `run_name`?

Reply:
Because it **args map dictionary with key name to the method argument, and during MLFlow design we decided following names for mlflow config variables:
 mlflow:
   experiment:
   run:
   client_id: 
   host:

Reply:
In this case we should make variable naming consistent.